external/FileSaver.1.3.3.min.js

```diff
@@ -0,0 +1,2 @@
+/*! @source http://purl.eligrey.com/github/FileSaver.js/blob/master/FileSaver.js */
```

Comment:
The attribution of this external library should IMO made more clearer (I guess this is the only condition MIT license gives). For example you could also copy its [LICENSE file](https://github.com/eligrey/FileSaver.js/blob/master/LICENSE.md) here into external directory. Also one can mention it in the README file or the about file.